src/main/resources/configurations.conf

```diff
@@ -25,4 +25,5 @@ workspace {
   submissionsPath="/workspaces/%s/%s/submissions"
   submissionByIdPath="/workspaces/%s/%s/submissions/%s"
   workflowOutputsByIdPath="/workspaces/%s/%s/submissions/%s/workflows/%s/outputs"
+  updateAttributesOnWorkspacePath="/workspaces/%s/%s/updateAttributes"
```

Comment:
Which endpoint is using this path? I don't see any usage of it in the rest of the code.